Autopilot/Test/Src/AttitudeManager/Test_Fsm.cpp

```diff
@@ -123,12 +123,9 @@ TEST(AttitudeManagerFSM, IfFetchInstructionsSucceedsTransitionToSensorFusion) {
 
 	attitudeManager attMng;
 
-	PMError_t PmNoError;
-	PmNoError.errorCode = 0;
-
 	/********************DEPENDENCIES*******************/
 
-	PM_GetCommands_fake.return_val = PmNoError;
+	GetCommands_fake.return_val = true;
```

Comment:
this is actually hoe we do C style function mocking. the GetCommands function is ours, but it's implementation uses freeRtos. In our unit test build system, we've got fff (https://github.com/meekrosoft/fff). 

At the top of this file, you'l notice 
![image](https://user-images.githubusercontent.com/32182266/117521752-6ed29780-af7d-11eb-834f-6b1852773eca.png)

That macro tells fff to create a fake function that gets called instead of the real implementation. Along with it, all these definitions get created to allow us to fake its behaviour. GetCommands_fake instructions fff to return this value next time the code calls this function.